src/cmdline.h

```diff
@@ -18,7 +18,7 @@ class CmdlineArgs {
     TCLAP::SwitchArg verbose;
     TCLAP::SwitchArg print_variables;
 
-    // NOTE: previously chkstat allowed multiple specifications of value
+    // NOTE: old chkstat allowed multiple specifications of value
```

Comment:
You left the chkstat name here, is this intentional?

Reply:
yes, because the old behaviour explicitly refers to the chkstat times